### ISSUE TITLE
can't add a new flag

### ISSUE THREAD

**Issue by prilaga** [halysongoncalves/pugnotification#27]
How to lock notification on status bar?
I using this configuration:

PugNotification.with(context).load()
                    .smallIcon(R.drawable.default_icon)
                    .title(push.title)
                    .autoCancel(true)
                    .message(push.alert)
                    .flags(Notification.DEFAULT_ALL | Notification.FLAG_AUTO_CANCEL)
                    .click(pIntent)
                    .simple().build();

But I can't to lock (fix) notification on status bar, so after swiping I can remove it.
This feature is working on my custom notification:

```
    NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
    builder
            .setSmallIcon(R.drawable.status_bar_icon)
            .setWhen(System.currentTimeMillis())
            .setAutoCancel(true)
            .setContentTitle(appName)
            .setContentText(text)
            .setLights(0xffffffff, 1000, 3000)
            .setPriority(Notification.PRIORITY_HIGH)
            .setContentIntent(pIntent);

    builder.setVibrate(new long[]{500, 500, 250, 250, 250, 250, 250});
    Notification notification = builder.build();

    if (LOCK_MODE)
    notification.flags |= Notification.FLAG_AUTO_CANCEL;

    NotificationManager manager = (NotificationManager)getSystemService(Context.NOTIFICATION_SERVICE);
    manager.notify(1, notification);
```

**Comment by prilaga** [comment]
I can't find how to add

.setOngoing(true)

**Comment by halysongoncalves** [comment]
Hi @OlegTarashkevich 

I have just released a new version (1.7.0) with this change. Now you can simply assign ongoing (true), that notification will no longer be removed.

Sorry for the delay! Anything can open new issues!

Thank you!
